docs/releases/3.2.4.txt

```diff
@@ -15,3 +15,7 @@ Bugfixes
 
 * Fixed a bug in Django 3.2 where a system check would crash on an abstract
   model (:ticket:`32733`).
+
+* Fixed a crash in Django 3.2 that could occur when running ``mod_wsgi`` with
+  the recommended settings whilst the Windows ``colorama`` library was
```

Comment:
```suggestion
  the recommended settings while the Windows ``colorama`` library was
```
see #12976 and dbdc192ca30b70a9f22c6bc4b3155bbccd01c323 :wink:

Reply:
> The lyf so short, the craft so long to lerne.

😀